src/bin/tui/status.rs

```diff
@@ -20,6 +20,7 @@ use cursive::traits::Identifiable;
 use cursive::view::View;
 use cursive::views::{LinearLayout, ResizedView, TextView};
 use cursive::Cursive;
+use std::borrow::Cow;
```

Comment:
Ooh! First use of `Cow` in grin.

Reply:
I do not really understand what a `Cow` is.

Reply:
Cursive accepts smth like AsRef<str> (don’t remember exactly), but we need to return some type, we went with String because in some cases we generate a status. But in some cases we return a static string, so what if we could avoid an extra allocation? Cow does exactly that - it keeps an owned T or borrowed T (until we mutate it, then it switches to the  owned variant, doing copy on write or cow). In our case we don’t mutate the content and lifetime for borrowed str is static.